src/tests/ignore_functions_test/vld.ini

```diff
@@ -181,4 +181,4 @@ SkipCrtStartupLeaks = yes
 ;   Valid Values: Functions names as strings separated by comma. Strings are case sensitive.
 ;   Default: None.
 ;
-IgnoreFunctionsList = GetOSVersion,SomeOtherString
\ No newline at end of file
+IgnoreFunctionsList = GetOSVersion,SomeOtherString,abcdefg,SomeOtherString,otherString,testOtherString
```

Comment:
>SomeOtherString [](http://example.com/codeflow?start=59&length=15)

consider another test without the ini that this does detect it as a leak. Or maybe the other test should ignore some functions but have at least 1 not ignored that gets detected.